rsconnect/certificates.py

```diff
@@ -0,0 +1,37 @@
+from pathlib import Path
+
+BINARY_ENCODED_FILETYPES = [".cer", ".der"]
+TEXT_ENCODED_FILETYPES = [".ca-bundle", ".crt", ".key", ".pem"]
+
+
+def read_certificate_file(location: str):
```

Comment:
If we are changing cacert to location then we need to be internally consistent and change existing cacert to something like cacert_location.

Reply:
Going down that route can potentially break backwards compatibility so that's an important aspect to consider.

Reply:
Hey @bcwu - could you help clarify your comment. Are you suggesting that I rename the variables named `cacert` to `cacert_location` so that the variable type is more obvious?

Reply:
I did a little more digging. It looks like `cafile` is the typical nomenclature for file name references. https://docs.python.org/3/library/ssl.html

Reply:
I was thinking about a couple of things:
1) Whether changing the type hints of cacert (from IO to str) will break any existing behavior
- In this case either way (type hint set to IO or str) is probably all right. In general it helps to think through whether changes are backwards compatible.
2) Possibly rename cacert if we do change the type hint
- this will probably cause more confusion than it clarifies. It will certainly break existing behavior; so, on second thought not a good idea, please disregard.

Reply:
I checked the usage of `cacert` in each of the methods where the type signature is changed. In each case the variable is only used in combination with `read_certificate_file`. So, this shouldn't break any exisiting behavior.